detection_rules/kbwrap.py

```diff
@@ -71,6 +72,36 @@ def upload_rule(ctx, rules, replace_id):
     return results
 
 
+@kibana_group.command('import-rules')
+@click.argument('rules', nargs=-1, required=True)
```

Comment:
because so many of the problems we hear about `upload-rule` is around validation, this takes the dict version of the rule, leaving the onus on the caller to present it however they choose.

With rule loader

```python
rc = RuleCollection.default()
rules = [r.contents.to_api_format() for r in rc.rules]
```

Which doesn't work well for a command.

This reinforces the idea of decoupling the file dict loader from `RuleCollection`, to allow dict load vs full `Rule` object loading, because then we can use `multi_collection` without the `Rule`

Reply:
I think I am going to consider this a blocker, because without it, the commands just don't make sense as is (without forcing an intermediate step of saving the rules in their final format and then loading them from files vs direct loading)

Reply:
uses the `multi_collection` loader. Will expand that to allow custom rules in the DAC feature branch